src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
```

Comment:
Does this test work? Do we have 'learn react' link on the App page?

Reply:
this is from when we create react app the default

Reply:
Right, so currently 'npm test' fails, either remove the test, or replace it with a working test.